src/lpython/parser/tokenizer.re

```diff
@@ -268,7 +270,7 @@ int Tokenizer::lex(Allocator &al, YYSTYPE &yylval, Location &loc, diag::Diagnost
             int_bin = "0"[bB][01]+;
             int_hex = "0"[xX][0-9a-fA-F]+;
             int_dec = digit+ (digit | "_" digit)*;
-            char =  [a-zA-Z_];
+            char = L | "_";
             name = char (char | digit)*;
```

Comment:
Would this accept a Hindi numeral? It seems it would only accept digits 0-9, but not Hindi numerals.

Reply:
I just added the support for Unicode numerals in the latest commit.